docs/internals/contributing/writing-documentation.txt

```diff
@@ -215,6 +216,31 @@ run the following command from the ``docs`` directory:
 The formatter will report any issues by printing them to the terminal and will
 reformat code blocks where possible.
 
+.. _documentation-lint-check:
+
+Documentation lint check
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Django's documentation is checked for reStructuredText style and formal issues
+using :pypi:`sphinx-lint`. This helps catch problems like stray tabs, trailing
+whitespace, excessive line length, and similar formatting problems.
+
+Once ``sphinx-lint`` is installed, the check can be run with the following
+command from the ``docs`` directory:
+
+.. console::
+
+     $ make lint
```

Comment:
Thanks so much for this tool. I spun it up today, and found I needed to install `enchant`, which makes sense. Do you think we should mention this in this doc? Or is it self-explanatory enough?

Reply:
Right, I meant that that python library requires the C enchant library as a system dependency, which is might be less clear for beginners, but looking at my shell scrollback I think this was when running `make check` not `make lint` after all, so disregard!